src/Tribe/Views/V2/View.php

```diff
@@ -942,12 +956,21 @@ protected function setup_repository_args( \Tribe__Context $context = null ) {
 
 		$context_arr = $context->to_array();
 
-		return [
-			'posts_per_page' => $context_arr['events_per_page'],
-			'paged' => max( Arr::get_first_set( array_filter( $context_arr ), [ 'paged', 'page' ], 1 ), 1 ),
-			'search' => $context->get( 'keyword', '' ),
+		$args = [
+			'posts_per_page'       => $context_arr['events_per_page'],
+			'paged'                => max( Arr::get_first_set( array_filter( $context_arr ), [
+				'paged',
+				'page',
+			], 1 ), 1 ),
```

Comment:
This is sort of hard to read/parse. ~Perhaps move the `max()` up above the args rather than doing in-place?~

```php
$paged = max(
  Arr::get_first_set(
    array_filter( $context_arr ),
    [
      'paged',
      'page',
    ],
    1
  ),
  1
);
```

Actually, that's pretty darn hard to read as well. Maybe it doesn't matter all that much. Pick your poison.